### DISCUSSION TITLE
Beatmap attributes not practical

### DISCUSSION THREAD

**Discussion by AosUK** [ppy/osu#24946] [Ideas]
While using the difficulty adjust mod, the values selected for the OD, AR etc are later changed by DT and possibly other mods. Meaning there isn't a way to easily see the actual information for the map you're playing. 
![img](https://github.com/ppy/osu/assets/134084802/bdd717e5-b9fa-4875-900f-5c0f0b3ccfa6)

**Comment by bdach** [comment]
yes, we know.

needs https://github.com/ppy/osu/pull/24642 (but with _much_ better execution) to happen first.
